test/integration/targets/delegate_to/test_loop_control.yml

```diff
@@ -0,0 +1,16 @@
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Test delegate_to with loop_control
+      ping:
+      delegate_to: "{{ item }}"
```

Comment:
this would be incorrect as the label is not 'item' but the value of `{{item}}`

Reply:
I am not sure I follow. The value of `{{ item }}` is `localhost` which is the host we're delegating to. The `item_var` is `item` and not `None` which is what this PR fixes. Am I missing something?

Reply:
'{{ item }}' should have no value unless it is the loop var, which you  'are changing' to 'localhost'

Reply:
I think I got confused, can you elaborate a bit more please? I was taking inspiration from the usage of `label` from http://docs.ansible.com/ansible/latest/playbooks_loops.html#loop-control, how do I use it differently?

Reply:
that uses it to resolve to the "item's name" in this case you are using it to resolve to the item itself and then try to use `item` again.

sooo, `loop_var: "{{item}}"' makes the loop var a var named `localhost` but you still try to use a var named `item` to delegate to.

Reply:
okay, agreed, but I am using `label` and not `loop_var`, so I need to use `"{{ item }}"` so the label is templated and displayed as `localhost` (and whatever other values are in `with_items`). If I were to use `loop_var`, it would be just the var name, so `item`.

The usage of `label` is redundant, but it triggers the bug so. Any ideas on how to write the test better? Thanks!